client/app/lib/request.coffee

```diff
@@ -18,7 +18,10 @@ exports.request = (type, url, data, callback) ->
         error: (data) ->
             fired = true
 
-            if callback? and data?
+            if data.getResponseHeader("X-Cozy-Login-Page")
```

Comment:
Shouldn't we test the presence of `data`? Something like `data?.getResponseHeader`